### PR TITLE
add common clang-format rules

### PR DESCRIPTION
**Pull request by FredKSchott**
- [X] CHANGELOG.md has been updated

***Only merge when there are no outstanding PRs, and remember to run `npm run format` after merging!***

/cc @rictic @justinfagnani

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -74,6 +75,7 @@
   "devDependencies": {
     "bower": "^1.7.9",
     "chai": "^3.5.0",
+    "clang-format": "^1.0.45",
```
Comment (rictic) [inline package.json]: pin to a specific version please

**Comment by FredKSchott** [comment]
I will rebase this off of master once the current batch of PRs are merged, and then commit the reformatting for a final look before merging.

**Comment by rictic** [comment]
Will there ever be a time when there's not a batch of open PRs?

**Comment by FredKSchott** [comment]
If we can get https://github.com/Polymer/polymer-cli/pull/490 in we should be good. Trying to avoid unnecessary work for @justinfagnani if we don't need to.

No harm in seeing it here though, and then rebasing later. Pushing formatted code up to this branch...

File: src/commands/help.ts [hunk 2]
```diff
@@ -21,6 +25,8 @@ const m = chalk.magenta;
 const CLI_TITLE = chalk.bold.underline('Polymer-CLI');
 const CLI_DESCRIPTION = 'The multi-tool for Polymer projects';
 const CLI_USAGE = 'Usage: \`polymer <command> [options ...]\`';
+
+// clang-format off - keep the custom white space here to help readibility
```
Comment (FredKSchott) [inline src/commands/help.ts]: looks like this is working now! Woo!

**Comment by FredKSchott** [comment]
@rictic PTAL

File: src/build/optimize-streams.ts [hunk 3]
```diff
@@ -1,8 +1,9 @@
-import {minify as uglify, MinifyOptions as UglifyOptions} from 'uglify-js';
 import {css as cssSlam} from 'css-slam';
```
Comment (rictic) [inline src/build/optimize-streams.ts]: Need to add the license header

File: src/commands/help.ts (hunk 2 above)
Comment (rictic) [inline src/commands/help.ts]: nice!

**Pull request merged**